### PR TITLE
Add support to kubevirt for virt-who-config

### PR DESCRIPTION
**Pull request by hkx303**
1. Add support to kubevirt for virt-who-config
2. Add support to Interval for Satellite New features. (support to chose 1440,2880,4320)
3. Add support to Get List of virt-who configurations per organization

### CODE REVIEW

**Comment by hkx303** [comment]
# pytest test_entities.py 
platform linux -- Python 3.7.3, pytest-4.0.2, py-1.8.0, pluggy-0.12.0
rootdir: /home/hkx303/Documents/CI/nailgun, inifile:
plugins: services-1.3.1, forked-1.0.2, mock-1.10.4, cov-2.7.1, xdist-1.29.0
collected 159 items                                                                 

test_entities.py ............................................................ [ 100%]

====================== 159 passed, 79 warnings in 1.46 seconds ======================

**Comment by hkx303** [comment]
Test:
[in]:
vhc = entities.VirtWhoConfig().search(
query={'search': 'name="esx"'}
)[0].read_per_organization()

[out]:
{'total': 2, 'subtotal': 2, 'page': 1, 'per_page': 20, 'search': None, 'sort': {'by': None, 'order': None}, 'results': [{'name': 'kubevirt', 'interval': 60, 'organization_id': 1, 'whitelist': '', 'blacklist': '', 'hypervisor_id': 'hostname', 'hypervisor_type': 'kubevirt', 'hypervisor_server': '1111', 'hypervisor_username': '', 'debug': True, 'satellite_url': 'ent-02-vm-03.lab.eng.nay.redhat.com', 'proxy': '', 'no_proxy': '', 'status': 'out_of_date', 'last_report_at': '2019-08-01 07:08:39 UTC', 'out_of_date_at': '2019-08-01 08:08:39 UTC', 'filtering_mode': 1, 'id': 9}, {'name': 'esx', 'interval': 60, 'organization_id': 1, 'whitelist': 'host-91', 'blacklist': '', 'hypervisor_id': 'hwuuid', 'hypervisor_type': 'esx', 'hypervisor_server': '10.73.3.198', 'hypervisor_username': 'administrator@vsphere.local', 'debug': False, 'satellite_url': 'ent-02-vm-03.lab.eng.nay.redhat.com', 'proxy': 'example.com:3228', 'no_proxy': '', 'status': 'out_of_date', 'last_report_at': '2019-08-01 07:50:54 UTC', 'out_of_date_at': '2019-08-01 08:50:54 UTC', 'filtering_mode': 1, 'id': 10}]}

File: nailgun/entities.py [hunk 1]
```diff
@@ -8008,3 +8018,22 @@ def read(self, entity=None, attrs=None, ignore=None, params=None):
             ignore = set()
         ignore.add('hypervisor_password')
         return super(VirtWhoConfig, self).read(entity, attrs, ignore, params)
+
+    def read_per_organization(self, synchronous=True, **kwargs):
```
Comment (jyejare) [inline nailgun/entities.py]: Rename this to 'get_organization_configs' or simillar.

Comment (hkx303) [reply nailgun/entities.py]: Ok, make sense, thank you

**Comment by eko999** [comment]
@hkx303 , please drop this test result since there are some private message.

**Comment by ntkathole** [comment]
Can one of the admins verify this patch?

**Pull request merged**